### PR TITLE
Added -n option

### PR DESCRIPTION
**Pull request by migperfer**
Limits the number of downloads to -n, sorting from most recent creation date to older

### CODE REVIEW

File: scdl/scdl.py [hunk 1]
```diff
@@ -371,6 +372,7 @@ def download_playlist(playlist):
     playlist_name = playlist['title'].encode('utf-8', 'ignore')
     playlist_name = playlist_name.decode('utf8')
     playlist_name = ''.join(c for c in playlist_name if c not in invalid_chars)
+    # Sort playlist by date of track creation
```
Comment (flyingrub) [inline scdl/scdl.py]: Is this needed here ?

Comment (jajaxdlol) [reply scdl/scdl.py]: No! My bad I forgot that comment there, feel free to remove it :)

**Comment by jajaxdlol** [comment]
I removed the misplaced comment, feel free to merge if you want :)

**Comment by natestemen** [comment]
This PR seems ready to go, and less changes than https://github.com/flyingrub/scdl/pull/283, so could we get this merged?

**Comment by flyingrub** [comment]
Merged :tada:

**Pull request merged**